README.md

```diff
@@ -12,6 +12,15 @@ Setting up
 4. Go to `localhost:3000/users/<vine-user-id>/atom` or 
    `localhost:3000/users/<vine-user-id>/rss`
 
+Usage with systemd
+------------------
+
+1. Copy `vinefeed.service` and `vinefeed.socket` to a systemd units directory, e.g. `/usr/lib/systemd/system` or `~/.config/systemd/user`
```

Comment:
I assume a symlink would be fine, especially when only used in the user instance. However, the binary has to be available at `/usr/bin/vinefeed`, so I noted about installation, too.